justfile

```diff
@@ -1,4 +1,4 @@
-PROJECTS := "pytest-saas pytest-itde"
+PROJECTS := "pytest-itde"
```

Comment:
Why remove it from the project list?

Reply:
I've been investigating the test failure.